content/en/docs/examples/virtual-machines/multi-network/index.md

```diff
@@ -30,118 +30,122 @@ bare metal and the clusters.
+        vm-network:
+
+EOF
     {{< /text >}}
 
-1. For a simple setup, deploy Istio control plane into the cluster
-
-        {{< text bash >}}
-        $ istioctl install \
-            -f manifests/examples/vm/values-istio-meshexpansion.yaml
-        {{< /text >}}
-
-    For further details and customization options, refer to the
-    [installation instructions](/docs/setup/install/istioctl/).
-
-### Setting up the VM
-
-Next, run the following commands on each machine that you want to add to the mesh:
-
-1.  Copy the previously created `cluster.env` and `*.pem` files to the VM. For example:
+1. Install with the virtual-machine features enabled:
 
     {{< text bash >}}
-    $ export GCE_NAME="your-gce-instance"
-    $ gcloud compute scp --project=${MY_PROJECT} --zone=${MY_ZONE} {key.pem,cert-chain.pem,cluster.env,root-cert.pem} ${GCE_NAME}:~
+$ istioctl install -f ./vmintegration.yaml
     {{< /text >}}
 
-1.  Install the Debian package with the Envoy sidecar.
-
-    {{< text bash >}}
-    $ gcloud compute ssh --project=${MY_PROJECT} --zone=${MY_ZONE} "${GCE_NAME}"
-    $ curl -L https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb > istio-sidecar.deb
-    $ sudo dpkg -i istio-sidecar.deb
-    {{< /text >}}
+#### Specify the network for the VM sidecar
 
-1.  Add the IP address of Istio gateway to `/etc/hosts`. Revisit the [preparing the cluster](#preparing-the-kubernetes-cluster-for-vms) section to learn how to obtain the IP address.
-The following example updates the `/etc/hosts` file with the {{< gloss >}}Istiod{{< /gloss >}} address:
+When following the [Virtual Machine Installation](/docs/setup/install/virtual-machine/) setup guide for creating the `sidecar.env` file we need to tweak the installation by adding the following entry:
 
     {{< text bash >}}
-    $ echo "${GWIP} istiod.istio-system.svc" | sudo tee -a /etc/hosts
+$ echo "ISTIO_META_NETWORK=vm-network" >> sidecar.env
     {{< /text >}}
 
-   A better options is to configure the DNS resolver of the VM to resolve the address, using a split-DNS server. Using
-   /etc/hosts is an easy to use example. It is also possible to use a real DNS and certificate for Istiod, this is beyond
-   the scope of this document.
-
-1.  Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs/`.
-
-    {{< text bash >}}
-    $ sudo mkdir -p /etc/certs
-    $ sudo cp {root-cert.pem,cert-chain.pem,key.pem} /etc/certs
+#### Create Gateway for application traffic
+
+The last step is to create a `Gateway` resource that allows application traffic from the VMs to route correctly.
+
+    {{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: cluster-aware-gateway
+  namespace: istio-system
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 443
+      name: tls
+      protocol: TLS
+    tls:
+      mode: AUTO_PASSTHROUGH
+    hosts:
+    - "*.local"
```

Comment:
AFAIK, this is already generated by setting `spec.values.global.multiCluster.enabled: true` in the `IstioOperator`, aside from that: 

1. That `Gateway` listens on `15443` rather than `443`
2. That `Gateway` listens on `*.global` rather than `*.local`

Reply:
FYI [new multicluster docs for 1.8](https://docs.google.com/document/d/1EZeTaoFb--Rr4EpJsjMBnRtm1VZT14mJpRbPJiJh37M/edit#heading=h.qspvl9cgiq2g) are going away from using `.global` and just sticking with `.cluster.local`